addons/mod_tool/dock.tscn

```diff
@@ -219,22 +219,22 @@ margin_bottom = -4.0
 [node name="ScrollContainer" type="ScrollContainer" parent="TabContainer/Mod Manifest"]
 margin_left = 7.0
 margin_top = 7.0
-margin_right = 1925.0
```

Comment:
I think we are ping-ponging these margins back and forth because of different screen resolutions.. 
But I think that only updates if something else in the scene was updated - in this case, which one of the tabs is visible

Reply:
yep that's a bit annoying, if we didn't change something in the scene, we can just discard the changes before committing.